src/commands/cmd_stream.cc

```diff
@@ -31,6 +31,7 @@
 #include "status.h"
 #include "time_util.h"
 #include "types/redis_stream.h"
+#include "types/redis_string.h"
```

Comment:
seems we acutually don't need it.

Reply:
sorry bout that, thought i would need it for the bulk string